extension/js/background_page/migrations.ts

```diff
@@ -38,13 +40,27 @@ const addKeyInfoFingerprints = async () => {
 };
 
 export const migrateGlobal = async () => {
-  const globalStore = await GlobalStore.get(['key_info_store_fingerprints_added']);
+  const globalStore = await GlobalStore.get(['key_info_store_fingerprints_added', 'local_drafts']);
   if (!globalStore.key_info_store_fingerprints_added) {
     console.info('migrating KeyStorage to add fingerprints and emails of each key...');
     await addKeyInfoFingerprints();
     await GlobalStore.set({ key_info_store_fingerprints_added: true });
     console.info('done migrating');
   }
+  // migrate local drafts (#3337)
+  const storageLocal = await storageLocalGetAll();
+  if (typeof globalStore.local_drafts === 'undefined') {
+    globalStore.local_drafts = {};
+  }
+  for (const key of Object.keys(storageLocal)) {
+    if (key.startsWith('local-draft-')) {
+      console.log(key);
+      console.info(`migrating local draft ${key}`);
+      globalStore.local_drafts[key] = storageLocal[key] as GmailRes.GmailDraftGet;
+      await storageLocalRemove([key]);
```

Comment:
Perhaps, it'd be better to remove old drafts after successful saving to GlobalStore?

Reply:
Agree, done 👍